js/views/contentScreen.js

```diff
@@ -45,12 +46,12 @@ var ContentScreen = React.createClass({
 
     return (
       <div className={this.props.screenClassName}>
+        <Watermark {...this.props} controlBarVisible={false} nonClickable={true}/>
```

Comment:
why is it not clickable on those screens? how does the user determine click-ability?

an empty `general.watermark.clickUrl` param should also determine click-ability.